user-testing/Android.md

```diff
@@ -0,0 +1,56 @@
+# Android Checklist
+
+## Preparation
+
+**Alice** will be the tester. You need 
+- an E-Mail address 
+- and the release candidate build
+- on a phone.
+- a Thunderbird installation with Enigmail or TB-AC.
+
+**Bob** is the "sparring partner" of Alice. Can be a daily-use Delta Chat
+Desktop account. You need
+- an E-Mail address
+- Delta Chat Desktop version >= 0.840.0
```

Comment:
So far I now stated that you need 2 accounts for testing. They send each other some messages and invite each other into groups, which is fine.

I thought it's more useful if Bob can just be a daily-use account with a stable Delta Desktop version. The main advantage is that testers only need a PC and a phone.